client/src/plugins/deployment-tool/DeploymentDetailsModal.js

```diff
@@ -125,14 +149,27 @@ export default class DeploymentDetailsModal extends React.PureComponent {
     return this.lazilyCheckConnection(values);
   }
 
-  getEndpointConfigFields(authType) {
+  getEndpointConfigHint(values, errors) {
```

Comment:
This seems to be not quite covered by tests right now, maybe we can add some test cases for this?

Reply:
So now we test whether the hint is displayed.